data-fusion/s1_flooding_visualisation/README.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Visualising floods using Sentinel-1 overlaid on Sentinel-2 imagery
+parent: Data Fusion
+layout: script
+permalink: /data-fusion/s1_flooding_visualisation/
+nav_exclude: true
+---
+
+
+## Authors
+
+**Author of the datafusion script:**
+- William Ray & Maxim Lamare
+  
+## Evaluate and visualize  
+ - [EO Browser](https://sentinelshare.page.link/gi8N)
```

Comment:
We can now define this part in the front matter. Please have a look at the [example](https://github.com/sentinel-hub/custom-scripts/blob/main/contribute/example/index.md#adding-examples).

Reply:
does this work with data fusion?

Reply:
Okay so we can resolve? I will try the new method when I add some new Planetscope scripts

Reply:
Could you please also update the link to use the updated evalscript for the consistency? Thanks!

Reply:
done!